Extension/src/LanguageServer/ui.ts

```diff
@@ -56,32 +56,52 @@ interface ConfigurationStatus {
     priority: ConfigurationPriority;
 }
 
-const commandArguments: string[] = ['oldUI']; // We report the sender of the command
+enum LanguageStatusPriority {
+    First = 0,
+    High = 1,
+    Mid = 2,
+    Low = 3
+}
+
+const commandArguments: string[] = []; // We report the sender of the command
 
-export class OldUI implements UI {
+export class CurrentUI implements UI {
```

Comment:
"CurrentUI"?  Unless we think we'll need to implement another replacement soon or additional alternative classes for this UI (at which point, the interface would likely need to be updated/different anyway), the otherwise unused interface could be removed, and any unnecessary plumbing removed to reduce code that needs to be maintained.

Also (existing issue), "UI" (or OldUI, NewUI, CurrentUI) is woefully vague.  It tells someone unfamiliar with the code very little about what it is.  If/when any other bit of UI were needed for other purposes, it could be confusing.  How about naming it "LanguageStatusUI", or something similarly specific?

Reply:
I changed it to LanguageStatusUI